Hanan/style.css

```diff
@@ -0,0 +1,105 @@
+*{
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+nav{
+    position: relative;
+    background-color:whitesmoke;
+    width: 100%;
+    box-shadow: 0px 0px 30px  black;
+}
+.logo{
+   width: 35px;
+   height: 40px;
+   border-radius: 50%;
+   margin-left: 10px;
+   margin-top: 10px;
+   
+}
+.btn{
+color: rgba(0, 110, 255, 0.884);
+border: 1px solid rgba(0, 110, 255, 0.884);
+border-radius: 12px;
+padding:8px 18px 8px 18px ;
+float: right;
+margin: 10px;
+
+}
+.blue{
+color: white;
+background-color: rgba(0, 110, 255, 0.884);
+}
+main{
+    margin: 0 auto;
+    width: 60%;
+}
+.background{
+    width: 100%;
+    height: 400px;
+    background-repeat: no-repeat;
+    background-size:cover;
+    background-position: center center;
+    position: relative;
+}
+.profile{
+    display: inline-block;
+    position: absolute;
+    width: 120px;
+    height:100px ;
+    border: 2px solid black;
+    border-radius: 50%;
+    margin-top: -50px;
+    margin-left:15px ;
+}
+.name{
+    position: absolute;
+    margin-top:20px ;
+    margin-left:150px ;
+}
+.location{
+    width: 20px;
+    height: 20px;
+    margin-top: 10px;
+}
+hr{
+    margin-top: 80px;
+    
+}
+.follow{
+    
+    position: absolute;
+    width: 40px;
+    height:40px ;
+    border: 2px solid white;
+    border-radius: 50%; 
+    margin-top: 15px;   
+}
+.p1{
```

Comment:
I don't think its the best idea to use a class for each image, you could of don't that by adding all images inside one div and giving them (float: left) and it will work fine 😄